autoscripts/README.md

```diff
@@ -0,0 +1,8 @@
+These debhelper autoscripts were taken from:
+https://git.launchpad.net/ubuntu/+source/debhelper/tree/autoscripts?h=applied/12.10ubuntu1
+
+WARNING: Each file MUST end with a line break, otherwise the scripts into which
```

Comment:
That's inelegant. You could add a line break automatically if it isn't there already.

Reply:
Agreed. This was simply because I was replicating the original Perl implementation as closely as possible. I will change it.